docs/guides/hybrid-graphics.md

```diff
@@ -9,6 +9,17 @@ This has been tested on the MacBookPro16,1 and the MacBookPro15,1. The 15,3 and
 1. Resume after suspend is broken, as the GMUX (graphics multiplexer) doesn't connect the iGPU to the display after resuming. For this to be fixed, a Linux driver for `acpi:APP000B:GPUC:` needs to be written (macOS uses AppleMuxControl2.kext). The extra battery life may make this a worthwhile trade-off (about 3 hours to almost 6 hours on a MacBookPro16,1)
 2. If using `DRI_PRIME=1` on programs causes system crashes, with "[CPU CATERR](https://gist.github.com/Redecorating/956a672e6922e285de83fdd7d9982e5e#gistcomment-3719941)" problem reports in macOS, disable dynamic power management with the `amdgpu.dpm=0` kernel argument, or `echo high | sudo tee /sys/bus/pci/drivers/amdgpu/0000:??:??.?/power_dpm_force_performance_level`.
 3. If apple-set-os is loaded, the iGPU will control display brightness, and if the iGPU isn't the boot gpu, the i915 Intel graphics driver will not load, and the display brightness cannot be changed (The exception to this is sometimes when rebooting from macOS Recovery etc, i915 loads fine).
+4. Some users with  MacBookPro16.1 and amdgpu 5500m models have reported problems. Unable to shutdown normally(sudden fan noise), getting T2 chip reset and excessive high temperatures under normal conditions. Posible workarounds are:
+    1. Set amd dynamic power management from auto to low. `echo low | sudo tee /sys/bus/pci/drivers/amdgpu/0000:??:??.?/power_dpm_force_performance_level`
+
+        To apply the low level permanently:
+   
+        ```plain
+            create /etc/udev/rules.d/30-amdgpu-pm.rules
+            KERNEL=="card0", SUBSYSTEM=="drm", DRIVERS=="amdgpu", ATTR{device/power_dpm_force_performance_level}="low"
```

Comment:
card0 is not always the amdgpu, sometimes it will be card1, or probably anything up to card9 if you have a bunch of egpu's and have those cards create drm devices first.

Consider using the device, subsystem_vendor, subsystem_device attributes if you want to make it only apply to the internal amd gpu's (for 5300M those values are 7340, 106b, and 0210, respectively, and on 5600M mbp16,4 they would be 7360, 106b, 021e)

Reply:
It would better if we add way to find which card number is the AMD gpu.
It will be easier for users than find out the vendor number.
For example radeon-profile tool uses card0, so it must be a way.